br/pkg/lightning/config/config.go

```diff
@@ -681,6 +738,7 @@ type TikvImporter struct {
 	DuplicateResolution DuplicateResolutionAlgorithm `toml:"duplicate-resolution" json:"duplicate-resolution"`
 	IncrementalImport   bool                         `toml:"incremental-import" json:"incremental-import"`
 	KeyspaceName        string                       `toml:"keyspace-name" json:"keyspace-name"`
+	AddIndexBySQL       Bool                         `toml:"add-index-by-sql" json:"add-index-by-sql"`
```

Comment:
can we use `*bool`. odd to have a 3 value type named `Bool`

Reply:
`*bool` is also okay, will change it later.

Reply:
https://github.com/pingcap/tidb/pull/42136/commits/2c1cadfde3b4d3add88021ad5b0173dd5a209a7a